python/jupytergis_qgis/src/plugins.ts

```diff
@@ -138,11 +173,118 @@ const activate = async (
   QGSWidgetFactory.widgetCreated.connect(widgetCreatedCallback);
   QGZWidgetFactory.widgetCreated.connect(widgetCreatedCallback);
 
-  console.log('jupytergis:qgisplugin is activated!');
+  /**
+   * The command to export the current main area jGIS project to QGIS file ('.qgz').
+   *
+   * A popup opens to choose a filepath (local from the current jGIS file) if the
+   * filepath is not provided in args.
+   */
+  app.commands.addCommand(CommandIDs.exportQgis, {
+    label: `Export To Qgis${installed ? '' : ' (QGIS is required)'}`,
+    isEnabled: () =>
+      installed && tracker.currentWidget
+        ? tracker.currentWidget.context.model.sharedModel.editable
```

Comment:
It would feel more natural to me to define the command only if QGIS is installed on the back-end (also suggesting a label rename here):

```suggestion
    if (installed) {
      app.commands.addCommand(CommandIDs.exportQgis, {
        label: `Export To .qgz`,
        isEnabled: () =>
          tracker.currentWidget
            ? tracker.currentWidget.context.model.sharedModel.editable
  }
```